plugins/SitesManager/templates/index.html

```diff
@@ -1,11 +1,15 @@
-<div ng-controller="SitesManagerController">
+<div ng-controller="SitesManagerController" class="SitesManager">
 
-    <div ng-include="'plugins/SitesManager/templates/sites-manager-header.html'"></div>
+    <div ng-include="'plugins/SitesManager/templates/sites-manager-header.html?cb=' + cacheBuster"></div>
```

Comment:
I appended the cacheBuster to the URL's so the templates will be invalidated on any new release etc. This was not the case before.